src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,88 @@
+import cn from 'classnames';
+import React, { useContext, useState } from 'react';
+// import { DispatchVisibleItemsContext, TodosContext }
+import { DispatchVisibleItemsContext }
+  from '../GlobalStateProvider';
+import './style.css';
+import { Status } from '../../types/Status';
+// import { TodoItemType } from '../../types/TodoItemType';
+
+export const TodosFilter: React.FC = () => {
+  // const setVisible = useContext(DispatchVisibleItemsContext);
+  const setVisibleStatus = useContext(DispatchVisibleItemsContext);
+  // const allTodos = useContext(TodosContext);
+
+  const [allSelected, setAllSelected] = useState(true);
+  const [activeSelected, setActiveSelected] = useState(false);
+  const [completedSelected, setCompletedSelected] = useState(false);
```

Comment:
you don't need it

just get the current status from the context and use it here to know which status is currently selected